packages/uswds-core/src/js/utils/focus-trap.mjs

```diff
@@ -1,8 +1,7 @@
-const assign = require("object-assign");
-const { keymap } = require("receptor");
-const behavior = require("./behavior");
-const select = require("./select");
-const activeElement = require("./active-element");
+import { keymap } from "receptor";
+import behavior from "./behavior.mjs";
+import select from "./select.mjs";
+import activeElement from "./active-element.mjs";
 
```

Comment:
This file uses `module.exports` instead of `export` like the other updated files. Is that intentional?

Reply:
Yes, good find. Fixed in c3bc2ceb203a7336d96fa0462fab58c56eda31fc.